src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -532,9 +532,28 @@ export class RoslynLanguageServer {
             childProcess = cp.spawn(serverPath, args, cpOptions);
         }
 
+        // Record the stdout and stderr streams from the server process.
```

Comment:
if the process failed to start it would previously silently timeout.  we weren't capturing stderr or erroring on process exit.